scene/multiplayer/multiplayer_spawner.cpp

```diff
@@ -97,12 +98,21 @@ void MultiplayerSpawner::add_spawnable_scene(const String &p_path) {
 	}
 	spawnable_scenes.push_back(sc);
 }
+
 int MultiplayerSpawner::get_spawnable_scene_count() const {
+	if (spawnable_scenes.is_empty()) {
+		return 0;
+	}
 	return spawnable_scenes.size();
 }
+
 String MultiplayerSpawner::get_spawnable_scene(int p_idx) const {
+	if (spawnable_scenes.is_empty()) {
+		return String();
+	}
 	return spawnable_scenes[p_idx].path;
 }
```

Comment:
It being empty isn't the sole cause of the crash, it's just an out of bounds access. 

```suggestion
String MultiplayerSpawner::get_spawnable_scene(int p_idx) const {
	ERR_FAIL_COND_V(p_idx < 0, "");
	ERR_FAIL_COND_V(p_idx >= spawnable_scenes.size(), "");
	return spawnable_scenes[p_idx].path;
}
```

Reply:
There are dedicated error macros for index checking:
```suggestion
String MultiplayerSpawner::get_spawnable_scene(int p_idx) const {
	ERR_FAIL_INDEX_V(p_idx, spawnable_scenes.size(), "");
	return spawnable_scenes[p_idx].path;
}
```

Reply:
This fails with int32 is not uint32.

Reply:
I think you just need to cast the size parameter. You'll lose the last bit though.

Reply:
There's an uint version index check.